src/btree/bt_cursor.c

```diff
@@ -2114,3 +2114,71 @@ __wt_btcur_close(WT_CURSOR_BTREE *cbt, bool lowlevel)
 
     return (ret);
 }
+
+/*
+ * __wt_btcur_bounds_row_position --
+ *     A unpositioned bounded cursor need to start its cursor next and prev walk from the lower or
+ *     upper bound depending on which direction it is going. This function calls cursor row search
+ *     to position the cursor appropriately.
+ */
+int
+__wt_btcur_bounds_row_position(
```

Comment:
I haven't been following along on how the project is broken down, so let me know if I'm asking for future scheduled work here...

Would it make sense to have a `__wt_btcur_bounds_position` here, and have it call `__wt_btcur_bounds_row_position` for row store, and return an `ENOTSUP` for other access methods?

Reply:
So we are planning to make that change in WT-9474! where we start implementing column store, which is https://github.com/wiredtiger/wiredtiger/pull/8080.

Reply:
Got it - thanks.